### ISSUE TITLE
metaQUEST: download reference genomes from a list

### ISSUE THREAD

**Issue by GiBacci** [ablab/quast#20]
I have a problem running metaQUAST with a set of reference genomes. In the metaQUAST paper it is reported that:

> The acquired list of species names can be fed to MetaQUAST in a plain text format, making it download the specified sequences from the NCBI database and use them for the reference based evaluation

I have a list of species obtained with Metaphlan2 and I would like to use them as reference for contig evaluation. I have tried several ways to do that using the `-R` option but unfortunately I received always the same error:

> Reference(s):
    WARNING: Skipping species.txt because it contains non-ACGTN characters.
  All references combined in combined_reference.fasta

So, basically, metaQUAST is searching for a set of sequence files and obviously it crashes since it can't find them. My species file is formatted as follows:

> genus species 1
   genus species 2
   genus species 3
   ...

Is there a way to make metaQUAST download all references reported in this file?

Thanks in advance,

cheers,

Giovanni

**Comment by almiheenko** [comment]
Hi Giovanni,

Sorry for the inconvience! This function has not properly described in the manual yet. We will fix it in the upcoming release. 
The file with list of species should be used with option `--references-list` instead of `-R`.

Best regards,
Alla
